src/seismometer/report/fairness.py

```diff
@@ -210,7 +210,7 @@ def fairness_table(
         )
         cohort_ratios = cohort_data.div(cohort_data.loc[cohort_data[COUNT].idxmax()], axis=1)
 
-        cohort_icons = cohort_ratios.drop(COUNT, axis=1).applymap(
+        cohort_icons = cohort_ratios.drop(COUNT, axis=1).map(
```

Comment:
![image](https://github.com/user-attachments/assets/7c52cdd2-35b6-4539-b924-8db07cd0110f)

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.map.html

mentions that map was added in Pandas 2.10

Reply:
rebased and dropped that commit